### DISCUSSION TITLE
Add current state for provisioing/alert-rules

### DISCUSSION THREAD

**Discussion by laa88rf** [grafana/grafana#63148] [Alerting]
Hello.

Please add information about the current state (Firing/Pending/Normal) in **/api/v1/provisioning/alert-rules** with all information.
Thank you.

**Comment by gillesdemey** [comment]
Hi! The provisioning API is not concerned with the actual status of an alert, it only manages the definition of alert rules before they are being sent to the evaluation engine so this is not possible.
